Swift/BRCrypto/BRCryptoSystem.swift

```diff
@@ -393,7 +393,7 @@ public final class System {
         self.listener  = listener
         self.account   = account
         self.onMainnet = onMainnet
-        self.path  = path
+        self.path  = path + (path.last == "/" ? "" : "/") + account.fileSystemIdentifier
```

Comment:
Do we think we'd ever support an environment where '/' isn't the (primary) separator (i.e. Windows)? Does Swift have something like Python's `os.path.sep` (that abstracts the actual value)?

Really, really small, likely moot question. Not meant to hold up review at all. Only change if you think it is worth it.